javascript/javascript1/week4/voice-assistant.js

```diff
@@ -0,0 +1,53 @@
+let userName = "nuha";
+let todo = [];
+let date = new Date("2020-06-01");
+var options = {
+  day: "numeric",
+  month: "long",
+  year: "numeric",
+};
```

Comment:
I liked this. This is the best way that I wished to see at other homework. Just one thing to get the current date use only `new Date()` so this code will be like this
```js
const today = new Date();
var options = {
  day: "numeric",
  month: "long",
  year: "numeric",
};
```